src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Books from './pages/Books';
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+      </Routes>
```

Comment:
- [ ] Kindly add the route for the Categories page as well as there are two pages required to have a route for the project. 👍